com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -407,8 +407,12 @@ private void Init(bool server)
                         NetworkLog.LogWarning($"{nameof(NetworkPrefab)} cannot be null ({nameof(NetworkPrefab)} at index: {i})");
                     }
 
-                    // Provide the name of the prefab with issues so the user can more easily find the prefab and fix it
-                    UnityEngine.Debug.LogWarning($"{nameof(NetworkPrefab)} (\"{NetworkConfig.NetworkPrefabs[i].Prefab.name}\") will be removed and ignored.");
+                    // Don't try to name the prefab if it doesn't exist
+                    if (NetworkConfig.NetworkPrefabs[i] != null && NetworkConfig.NetworkPrefabs[i].Prefab != null)
```

Comment:
huh? we are already in `if (NetworkConfig.NetworkPrefabs[i] == null || NetworkConfig.NetworkPrefabs[i].Prefab == null)` scope by now, see line 405
probably, there should be a better fix or logic flow here.

Reply:
Yeah... it doesn't need to be there at all.  Removed it completely.

Reply:
what issue this was causing? I'm just curious.